arch/arm/boot/dts/aspeed-bmc-opp-vesnin.dts

```diff
@@ -232,3 +232,16 @@
 &wdt2 {
 	aspeed,alt-boot;
 };
+
+&sdmmc {
+	status = "okay";
+};
+
+&sdhci1 {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_sd2_default>;
+	cd-inverted;
+	disable-wp;
```

Comment:
Add a comment "Vesnin BMC uses microSD, there is no WP switch, ignore it". Or at least mention it in the Analysis section in JIRA.